docs/DesignTokens/Taxonomy.stories.mdx

```diff
@@ -0,0 +1,138 @@
+- Scale — Choices within a predefined set of values, like varying size, space, shade, and other options.
+  - Ordinal: t-shirt sizes (xxs, xs, sm, md, lg, xl, xxl…).
+  - Sequential/stepped/numerical: base 100 (100, 200, 300…), base 1/index (0_5, 1, 2, 4…)
+  - Proportional: -1x, 0_5x, 1x, 2x…
+  - Keyword/Named:
+    - circle
+    - regular, tight, loose
+    - regular, medium, semi-bold, bold
+    - sans, serif, mono
+    - minimum
+    - ease-in, ease-out
+- Mode — Distinguish values across two or more surface/background settings on which elements appear. Use only when there’s a distinction (e.g., color and shadow for dark theme).
+  - `on-light`, `on-dark`, `on-light-hc`, `on-dark-hc`, or any colored surface like `invert`, `on-accent`, `on-primary`, `on-secondary`, `on-surface`.
+
+### Other naming rules
+
+- For compound words, use hyphens to separate the words: `line-height`, `on-primary`, `on-dark-hc`
+- For decimal values, replace the decimal point with an underscore: `0_5` for 0.5.
+- For negative values, use a hyphen: `-1`.
+- “Avoid dogmatically including all levels possible or duplicating token-tuples redundantly. Instead, include only the levels needed to sufficiently describe and distinguish purposeful intent.”
+
+Refer to [Naming Tokens in Design Systems](https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676) by Nathan Curtis for further examples.
+
+Exceptions to the rules described in Nathans's article:
+
+- Property comes before Concept and Sub-concept.
+
+## Resources
+
+### Design tokens taxonomy
+
+- [Material Design 3 Design Tokens](https://m3.material.io/foundations/design-tokens/overview), by Google.
+- [Naming Tokens in Design Systems](https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676), by Nathan Curtis.
+- [Naming Tokens in Design Systems](https://www.figma.com/community/file/1096728307713953219), Figma template by @mconde for Nathan Curtis article.
+- [The many faces of themeable design systems](https://bradfrost.com/blog/post/the-many-faces-of-themeable-design-systems/), by Brad Frost. Managing the different flavors of themeability including brands, sub-brands, white-labeling, themes, and more.
+- [Naming design tokens](https://uxdesign.cc/naming-design-tokens-9454818ed7cb), by Lukas Opperman.
+- [A design token taxonomy for Checkout.com](https://www.gerireid.com/design-token-taxonomy.html), by Geri Reid.
+
+More on contextual modifiers (e.g., `on-accent`, `on-primary`):
+
+- [Minimalist tokens](https://codepen.io/mikemai2awesome/pen/bGMpKWQ/9aa5fa652ea0f810a9649255a60acd26), on Codepen.
+- [Stateful theming](https://speakerdeck.com/csswizardry/4half-methods-for-theming-in-s-css?slide=32)), by Harry Roberts.
+
+### Standards
+
+- [Design Tokens Format Module](https://design-tokens.github.io/community-group/format/) — Design Tokens specification draft by the W3C Design Tokens Community Group.
```

Comment:
I don't think this ought to be in the public documentation we provide with Storybook

Reply:
I just simplified/cleaned this up. 👍